database/factories/OrganizationSettingFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\{
+    OrganizationSetting,
+    User
+};
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class OrganizationSettingFactory extends Factory
+{
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = OrganizationSetting::class;
+
+    /**
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'point_of_contact_id' => 1
```

Comment:
I had to set to 1. When I used the User Factory I was getting a weird duplication issue when testing.